leakcanary-android-core/src/main/java/leakcanary/DefaultOnHeapAnalyzedListener.kt

```diff
@@ -55,6 +73,50 @@ class DefaultOnHeapAnalyzedListener(private val application: Application) : OnHe
     )
   }
 
+  /**
+   * Android TV devices do not have notifications, therefore the only easy and non-invasive way
+   * to communicate with user is via Toast messages. These are used just to grab user attention and
+   * to direct them to Logcat where a much more detailed report will be printed.
+   */
+  private fun showToast(heapAnalysis: HeapAnalysis) {
+    // Post the Toast into main thread and wrap it with try-catch in case Toast crashes (it happens)
+    handler.post {
+      val toastText: String = when (heapAnalysis) {
+        is HeapAnalysisSuccess -> {
+          val leakTypeCount = heapAnalysis.applicationLeaks.size + heapAnalysis.libraryLeaks.size
+          application.getString(
+              R.string.leak_canary_tv_analysis_success,
+              leakTypeCount,
+              heapAnalysis.libraryLeaks.size
+          )
+        }
+        is HeapAnalysisFailure -> application.getString(R.string.leak_canary_tv_analysis_failure)
+      }
+      Toast.makeText(
+          application,
+          toastText,
+          Toast.LENGTH_LONG
+      ).show()
+    }
+  }
+
+  /**
+   * Android TV with API 26+ has a bug where launcher icon doesn't appear, so users won't know how
+   * to launch LeakCanary Activity.
+   * This method would print an adb command that launched LeakCanary into the logcat
+   */
+  private fun printIntentInfo() {
+    val leakClass = LeakActivity::class.java
+    val message = """====================================
+ANDROID TV LAUNCH INTENT
+====================================
+Run the following adb command to display the list of leaks:
+
+adb shell am start -n "${application.packageName}/${leakClass.name.replace(leakClass.simpleName, "LeakLauncherActivity")}"
```

Comment:
`leakClass.name.replace(leakClass.simpleName, "LeakLauncherActivity")` 

Isn't that `leakClass.package.name + "LeakLauncherActivity"` ?